abdulfatai360/banka
# #165049208 Set up automated testing framework

#### What does this PR do?
Setup the project's automated testing framework

#### Description of Task to be completed?
- Have Mocha installed as the testing framework
- Have ESLint plugin for Mocha installed
- Update `.eslintrc` file to add configurations for Mocha

#### How should this be manually tested?
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto our local machine
- Navigate to the repo by running `cd banka/` on terminal
- Ensure your working directory is clean by running  `git status`  command on terminal
- Get a copy of the PR by typing  `git fetch origin pull/17/head:ch-setup-testing-framework-165049208`
- Run `npm install` in terminal to install all project's dependencies onto your local machine
- Now that you have a copy of the branch, switch to it using  `git checkout ch-setup-testing-framework-165049208`
- Your directory will now be an exact copy of the PR.

#### What are the relevant pivotal tracker stories?
#165049208
